Changes

```diff
@@ -57,6 +57,9 @@ Working version
 - GPR#2308: More debugging information on [Cmm] terms
   (Mark Shinwell, review by Stephen Dolan)
 
+- GPR#7878: Replaced TypedtreeIter with tast_iterator
```

Comment:
Please also quote the relevant issue number #8539.

Reply:
Sure thing. How should I do this? I assume I shouldn’t replace 7878.

Reply:
Great PR! See [contributing](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#changelog) for reference on how to edit `Changes`.